### DISCUSSION TITLE
European Union Public License v1.1

### DISCUSSION THREAD

**Discussion by manticore-projects** [JSQLParser/JSqlParser#1661] [Ideas]
Maybe, additionally licensing the software under the new European Union Public License v1.1 would be possible, even when there is no direct impact in practice?

**Comment by wumpz** [comment]
Why? Is there any benefit? I do not know this license.

**Reply by manticore-projects** [reply]
Its a new license, published by the Europe Union for supporting European software development.

**Comment by wumpz** [comment]
So far I understand it, but why is Apache 2.0 not sufficient?

**Reply by manticore-projects** [reply]
It is sufficient, absolutely. It was only a suggestion to emphasize European software development.

**Comment by wumpz** [comment]
OK. Then I leave it for now.
